config/src/main/java/com/alibaba/nacos/config/server/constant/Constants.java

```diff
@@ -142,7 +142,9 @@
     public static final String LINE_SEPARATOR = Character.toString((char) 1);
     
     public static final String WORD_SEPARATOR = Character.toString((char) 2);
-    
+    /**
```

Comment:
The CI failed maybe due to missing the line feed here.

Reply:
Thanks for your reminder and I will fix it.